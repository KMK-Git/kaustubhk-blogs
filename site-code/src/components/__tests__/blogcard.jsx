import React from 'react';
import renderer from 'react-test-renderer';

import BlogCard from '../blogcard';

const { act } = renderer;

describe('BlogCard', () => {
  it('renders correctly', async () => {
    let tree;
    act(() => {
      tree = renderer
        .create(<BlogCard>Testing</BlogCard>);
    });
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
