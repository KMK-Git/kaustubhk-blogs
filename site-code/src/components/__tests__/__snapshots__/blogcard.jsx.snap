// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogCard renders correctly 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-y1gt6f{margin-top:40px;}",
      }
    }
    data-emotion="css y1gt6f"
  />,
  <div
    className="MuiBox-root css-y1gt6f"
  >
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-11lq3yg-MuiGrid-root{box-sizing:border-box;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-flex-wrap:wrap;-webkit-flex-wrap:wrap;-ms-flex-wrap:wrap;flex-wrap:wrap;width:100%;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;}",
        }
      }
      data-emotion="css 11lq3yg-MuiGrid-root"
    />
    <div
      className="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    >
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-1tbav1i-MuiGrid-root{box-sizing:border-box;margin:0;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;}@media (min-width:600px){.css-1tbav1i-MuiGrid-root{-webkit-flex-basis:8.333333%;-ms-flex-preferred-size:8.333333%;flex-basis:8.333333%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:8.333333%;}}@media (min-width:900px){.css-1tbav1i-MuiGrid-root{-webkit-flex-basis:16.666667%;-ms-flex-preferred-size:16.666667%;flex-basis:16.666667%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:16.666667%;}}@media (min-width:1200px){.css-1tbav1i-MuiGrid-root{-webkit-flex-basis:16.666667%;-ms-flex-preferred-size:16.666667%;flex-basis:16.666667%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:16.666667%;}}@media (min-width:1536px){.css-1tbav1i-MuiGrid-root{-webkit-flex-basis:16.666667%;-ms-flex-preferred-size:16.666667%;flex-basis:16.666667%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:16.666667%;}}",
          }
        }
        data-emotion="css 1tbav1i-MuiGrid-root"
      />
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-0 MuiGrid-grid-sm-1 MuiGrid-grid-md-2 css-1tbav1i-MuiGrid-root"
      />
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-1qf8bwp-MuiGrid-root{box-sizing:border-box;margin:0;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;-webkit-flex-basis:100%;-ms-flex-preferred-size:100%;flex-basis:100%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:100%;}@media (min-width:600px){.css-1qf8bwp-MuiGrid-root{-webkit-flex-basis:83.333333%;-ms-flex-preferred-size:83.333333%;flex-basis:83.333333%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:83.333333%;}}@media (min-width:900px){.css-1qf8bwp-MuiGrid-root{-webkit-flex-basis:66.666667%;-ms-flex-preferred-size:66.666667%;flex-basis:66.666667%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:66.666667%;}}@media (min-width:1200px){.css-1qf8bwp-MuiGrid-root{-webkit-flex-basis:66.666667%;-ms-flex-preferred-size:66.666667%;flex-basis:66.666667%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:66.666667%;}}@media (min-width:1536px){.css-1qf8bwp-MuiGrid-root{-webkit-flex-basis:66.666667%;-ms-flex-preferred-size:66.666667%;flex-basis:66.666667%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:66.666667%;}}",
          }
        }
        data-emotion="css 1qf8bwp-MuiGrid-root"
      />
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10 MuiGrid-grid-md-8 css-1qf8bwp-MuiGrid-root"
      >
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-1xqfzky-MuiCard-root{overflow:hidden;background-color:#ffffff;}",
            }
          }
          data-emotion="css 1xqfzky-MuiCard-root"
        />
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-5u90k0-MuiPaper-root-MuiCard-root{background-color:#fff;color:rgba(0, 0, 0, 0.87);-webkit-transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;border-radius:4px;box-shadow:0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);overflow:hidden;background-color:#ffffff;}",
            }
          }
          data-emotion="css 5u90k0-MuiPaper-root-MuiCard-root"
        />
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-5u90k0-MuiPaper-root-MuiCard-root"
        >
          <style
            dangerouslySetInnerHTML={
              Object {
                "__html": ".css-46bh2p-MuiCardContent-root{padding:16px;}.css-46bh2p-MuiCardContent-root:last-child{padding-bottom:24px;}",
              }
            }
            data-emotion="css 46bh2p-MuiCardContent-root"
          />
          <div
            className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <p>
              Testing
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
